.gitignore

```diff
@@ -8,7 +8,6 @@ __pycache__/
 
 # Distribution / packaging
```

Comment:
I think we need to add some of the model outputs to the registry. I tried to pull this repo is at a GBs level.

Reply:
Yes, I agree! We talked about that in our meeting, but I think we can accept the PR as it is now, and leave that file reduction for another PR maybe.